code/controllers/subsystem/lua.dm

```diff
@@ -63,6 +47,23 @@ SUBSYSTEM_DEF(lua)
 		warning(e.name)
 		return time
 
+/datum/controller/subsystem/lua/OnConfigLoad()
+	// Get the current working directory - we need it to set the LUAU_PATH environment variable
+	var/here = world.shelleo(world.system_type == MS_WINDOWS ? "cd" : "pwd")[SHELLEO_STDOUT]
+	here = replacetext(here, "\n", "")
+	var/last_char = copytext_char(here, -1)
+	if(last_char != "/" && last_char != "\\")
+		here += "/"
+
+	// Read the paths from the config file
+	var/list/lua_path = list()
+	var/list/config_paths = CONFIG_GET(str_list/lua_path)
```

Comment:
This smells like the kind of thing we should protect. Can you protect globals?

Reply:
Configs are datums, therefore they can have `vv_var_edit` return `FALSE`.

Reply:
there is a var you set on the config entry datum and the global config override for vv_var_edit will handle returning false.